source/_posts/2016-11-08-Vapor-mysql.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Adding MySQL to your Vapor project
+date: 2016-11-08 21:31:21
+tags: vapor,mysql
+authorId: olha
+categories:
+- Vapor
+---
+
+In this tutorial we will walk through how to add a database layer to a [Vapor](vapor.codes) project. We will be using MySQL and will assume that you have set it up on your local machine [(guide here)](https://dev.mysql.com/doc/refman/5.6/en/osx-installation-pkg.html).
+
+We are continuing on our 'Car' project. You don't necacery have to have read that tutorial if you already are a Vapor rock star. If not then please find it [here](https://engineering.nodesagency.com/articles/Vapor/6-simple-steps-to-setup-vapor/).
+
+### 1. Adding the MySQL package
+
```

Comment:
I think it would be nice to mention how to install mysql on your computer, especially since it's only three commands and it could be much easier for people than to search the internet for it.

1. Install [Homebrew](http://brew.sh):   
`/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
2. Install MySQL:  
`brew install mysql`
3. Start the MySQL server:  
`mysql.server start`